exemplars_parent/exemplars/src/main/java/io/prometheus/client/exemplars/api/Value.java

```diff
@@ -0,0 +1,13 @@
+package io.prometheus.client.exemplars.api;
+
+/**
+ * Lazy value holder for a double value.
+ * <p/>
+ * Counters and Gauges track their current value using a DoubleAdder, so getting the value
```

Comment:
I changed the `CounterExemplarSampler` to this
```java
Exemplar sample(double increment, Value newTotalValue, Exemplar previous);
```
The default implementation uses the `increment` and ignores the `newTotalValue`. However, I still kept the `newTotalValue` available in the API so that users can access it if needed (e.g. start sampling after the total value exceeded a threshold).